understanding/20/page-titled.html

```diff
@@ -96,15 +96,23 @@ <h2>Examples of Page Titled</h2>
          <dt>An HTML web page</dt>
          <dd>The descriptive title of an HTML web page is marked up with the &lt;title&gt; element so
                that it will be displayed in the title bar of the user agent.</dd>
-         <dt>A document collection</dt>
+         <dt>A guide on authoring practices</dt>
```

Comment:
```suggestion
         <dt>A multi-page guide on authoring practices</dt>
```

Reply:
We still need the simple titular category of "a document collection" or "set of web pages"